debian/clearwater-diags-monitor.init.d

```diff
@@ -60,11 +60,16 @@ DAEMON=/usr/share/clearwater/bin/clearwater_diags_monitor
 [ -r /etc/default/$NAME ] && . /etc/default/$NAME
 
 # Load the VERBOSE setting and other rcS variables
-. /lib/init/vars.sh
+[ -r /lib/init/vars.sh ] && . /lib/init/vars.sh
 
 # Define LSB log_* functions.
 # Depend on lsb-base (>= 3.0-6) to ensure that this file is present.
 . /lib/lsb/init-functions
+[ "$(type -t log_daemon_msg)" = "function" ] || function log_daemon_msg { log_success_msg $@ ; }
+[ "$(type -t log_end_msg)" = "function" ] || function log_end_msg { true ; }
+
+# Include /etc/init.d/functions if available.
+[ -r /etc/init.d/functions ] && . /etc/init.d/functions
```

Comment:
Good question - possibly.  I think we should see what happens with the next init.d script, and commonize then if needed.